train_agent.py

```diff
@@ -26,10 +27,29 @@
 def main():
     print_config()
 
-    relative_paths = np.loadtxt(CONFIG['imagefile_path'], dtype=str)
-    images_base_path = os.path.dirname(CONFIG['imagefile_path'])
-    absolute_paths = [images_base_path + i.strip('.') for i in relative_paths]
-    bboxes = np.load(CONFIG['boxfile_path'], allow_pickle=True)
+    # Load dataset to initialize environment with
+    absolute_paths, bboxes = [], []
+    jsonfile_path = CONFIG.get('jsonfile_path')
+
+    if jsonfile_path:
+        print(f'Loading data from {jsonfile_path}')
+        images_base_path = os.path.dirname(jsonfile_path)
+        with open(jsonfile_path) as f:
+            data = json.load(f)
+            for example in data:
+                absolute_path = os.path.join(images_base_path, example['file_name'])
+                if os.path.exists(absolute_path):
+                    absolute_paths.append(absolute_path)
+                    bboxes.append(list(example['bounding_boxes']))
```

Comment:
It is unclear what the format of the bounding boxes is. Maybe we call to standard_bounding box here? Or is the env doing that anyway?